src/Tribe/Views/V2/Template/Page.php

```diff
@@ -270,7 +270,7 @@ public function should_hijack_page_template( \WP_Query $query ) {
 		}
 
 		// We wont hijack in case we are not dealing with a Post Type query
-		if ( ! in_array( TEC::POSTTYPE, (array) $query->get( 'post_type' ) ) ) {
+		if ( empty( $query->tribe_is_event_query ) ) {
```

Comment:
The flag property is set [in the `Tribe__Events__Query` class](https://github.com/moderntribe/the-events-calendar/blob/e95c1d91b4091e6865c6e7f9fd9cf179ee16b435/src/Tribe/Query.php#L186-L191), that will, eventually, be cycled out of v2.
I would re-implement this check in a Context location.

Reply:
We should revisit that and the method inside of `Bootstrap::should_load`. For now I think we can keep it.